ign_ros2_control/src/ign_system.cpp

```diff
@@ -376,9 +378,15 @@ bool IgnitionSystem::initSim(
       // independently of existence of command interface set initial value if defined
       if (!std::isnan(initial_position)) {
         this->dataPtr->joints_[j].joint_position = initial_position;
+        this->dataPtr->ecm->CreateComponent(
+          this->dataPtr->joints_[j].sim_joint,
+          sim::components::JointPositionReset({initial_position}));
```

Comment:
It should be ignition::gazebo::components::JointPositionReset({initial_position}));